src/ParameterStack.ts

```diff
@@ -59,5 +60,46 @@ export class ParameterStack extends Stack {
     });
 
   }
+  private rxMissionZgwParameters() {
+
+    new StringParameter(this, 'ssm-rxmission-zgw-1', {
+      parameterName: Statics.ssmRxMissionZgwZakenApiUrl,
+      stringValue: '-',
+    });
+
+    new StringParameter(this, 'ssm-rxmission-zgw-2', {
+      parameterName: Statics.ssmRxMissionZgwDocumentenApiUrl,
+      stringValue: '-',
+    });
+
+    new StringParameter(this, 'ssm-rxmission-zgw-3', {
+      parameterName: Statics.ssmRxMissionZgwZaaktype,
+      stringValue: '-',
+    });
+
+    new StringParameter(this, 'ssm-rxmission-zgw-4', {
+      parameterName: Statics.ssmRxMissionZgwZaakstatus,
+      stringValue: '-',
+    });
+
+    new StringParameter(this, 'ssm-rxmission-zgw-5', {
+      parameterName: Statics.ssmRxMissionZgwInformatieObjectType,
+      stringValue: '-',
+    });
+
+    new StringParameter(this, 'ssm-rxmission-zgw-6', {
+      parameterName: Statics.ssmRxMissionZgwClientId,
+      stringValue: '-',
+    });
+
+    new StringParameter(this, 'ssm-rxmission-zgw-7', {
```

Comment:
Dit is denk ik te vroeg, gezien mogelijk meerdere roltypes nodig zijn, pas param van maken als blijkt dat die nodig is / hoe veel?